product_docs/docs/net_connector/6.0.2.1/02_requirements_overview.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Requirements overview"
+---
+
+<div id="requirements_overview" class="registered_link"></div>
+
+These are the supported versions and platforms for the EDB .NET Connector.
+
+The EDB .NET Connector is certified with EDB Postgres Advanced Server version 10 and later.
+
+The EDB .NET Connector graphical installers are supported on the following Windows platforms:
+
+64-bit Windows:
+
+-   Windows Server 2019
+-   Windows Server 2016
+-   Windows Server 2012 R2
```

Comment:
I think Windows 2012 is EOL'ed and probably shouldn't be listed anymore

Reply:
Just noticed that OS version are also listed. The following are EOL'ed
Windows Server 2016
Windows Server 2012 R2
Windows 8.1 64 bit
Windows 8.1 32 bit

The only supported ones are the following
Windows Server 2019
Windows 10 64 bit
Windows 10 32 bit.